src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketInformation.cs

```diff
@@ -5,16 +5,16 @@ namespace System.Net.Sockets
 {
     public struct SocketInformation
     {
-        public byte[] ProtocolInformation { get; set; }
-        public SocketInformationOptions Options { get; set; }
+        public byte[] ProtocolInformation { readonly get; set; }
+        public SocketInformationOptions Options { readonly get; set; }
 
```

Comment:
Technically these aren't needed, the [docs state](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/struct#readonly-instance-members):

> The compiler declares a get accessor of an auto-implemented property as readonly, regardless of presence of the readonly modifier in a property declaration.

However, it isn't harmful or incorrect.